dist/zone_externs.js

```diff
@@ -0,0 +1,442 @@
+/**
+* @license
+* Copyright Google Inc. All Rights Reserved.
+*
+* Use of this source code is governed by an MIT-style license that can be
+* found in the LICENSE file at https://angular.io/license
+*/
+
+/**
+ * @fileoverview Externs for zone.js
```

Comment:
we can verify that these work in a real project, like overwrite https://github.com/alexeagle/closure-compiler-angular-bundling/blob/master/vendor/zone_externs.js with yours and re-build